CommonMark.Portable45/CommonMark.Portable45.csproj

```diff
@@ -0,0 +1,175 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm afraid same directory will break the VS2013 build due to packages.config. Maybe it's time to update the VS requirement in README.md?